### PR TITLE
[MT][browser] Disabling tests failing in the last two weeks on MT.

### PR DESCRIPTION
**Pull request by ilonatommy**
Continuation of https://github.com/dotnet/runtime/pull/93224.

Why do we remove test disabling and increase the thread for it instead:
Failure of type: https://github.com/dotnet/runtime/issues/91549 is caused by too small thread pool size in the browser, so we could avoid it by increasing it.
```
 System.Threading.Tasks.TaskSchedulerException : An exception was thrown by a TaskScheduler.
info: ---- System.Threading.ThreadStartException : Thread failed to start.
info: -------- System.ExecutionEngineException : mono_thread_platform_create_thread() failed
info:    at System.Threading.Tasks.Task.ScheduleAndStart(Boolean needsProtection)
info:    at System.Threading.Tasks.Task.Start(TaskScheduler scheduler)
info:    at System.Threading.Tasks.Task.Start()
info:    at System.Threading.Tasks.Tests.TaskPropertiesTests.TaskOptionTest(Boolean runSync)
info:    at System.Threading.Tasks.Tests.TaskPropertiesTests.TaskOptionsTestAsync()
info:    at System.Reflection.MethodBaseInvoker.InterpretedInvoke_Method(Object obj, IntPtr* args)
info:    at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
info: ----- Inner Stack Trace -----
info:    at System.Threading.Thread.ThrowThreadStartException(Exception ex)
info:    at System.Threading.Thread.StartCore()
info:    at System.Threading.Thread.Start(Object parameter, Boolean captureContext, Boolean internalThread)
info:    at System.Threading.Thread.UnsafeStart(Object parameter)
info:    at System.Threading.Tasks.ThreadPoolTaskScheduler.QueueTask(Task task)
info:    at System.Threading.Tasks.TaskScheduler.InternalQueueTask(Task task)
info:    at System.Threading.Tasks.Task.ScheduleAndStart(Boolean needsProtection)
info: ----- Inner Stack Trace -----
```

### CODE REVIEW

File: src/libraries/System.Threading.ThreadPool/tests/System.Threading.ThreadPool.Tests.csproj [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <TestRuntime>true</TestRuntime>
+    <_WasmPThreadPoolSize Condition="'$(MonoWasmBuildVariant)' == 'multithread'">64</_WasmPThreadPoolSize>
```
Comment (radical) [inline src/libraries/System.Threading.ThreadPool/tests/System.Threading.ThreadPool.Tests.csproj:6]: What is this for? Can you please add the reasoning to the PR description?

**Comment by radical** [comment]
cc @pavelsavara for review of the property value change.

**Review by pavelsavara — APPROVED** [review]

**Pull request merged**